### PR TITLE
Add indicators for gaps

### PR DESCRIPTION
**Pull request by aneshodza**
Added indicators and set the indicator to show on every gap
image

### CODE REVIEW

**Comment by Janis-Leuenberger** [comment]
This existed before?

File: app/helpers/timer_sessions_helper.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 module TimerSessionsHelper
   MAX_SUBJECT_LENGTH = 75
   SECONDS_IN_MINUTE = 60
-  GAP_LIMIT_IN_MINUTES = 5
+  GAP_LIMIT_IN_MINUTES = 1
```
Comment (Janis-Leuenberger) [inline app/helpers/timer_sessions_helper.rb:6]: Meaning it's no longer a gap than 5 minutes? I'd like that to be infinite.

Comment (aneshodza) [reply app/helpers/timer_sessions_helper.rb:6]: No its the other way around. If the gap is longer than that number it counts as a gap. Having a 1 minute gap marked makes more sense to me than only counting it form 5 minutes upwards

**Review by rnestler — APPROVED** [review]
We could discuss the color, but I don't care that much and love the feature. CI seems to need fixing?

**Comment by pandermatt** [comment]
@rnestler agree. I'm colorblind, so for me, it's hard to see 🥺. But I would love if it adds also a width of maybe 5px

File: app/helpers/timer_sessions_helper.rb (hunk 1 above)
Comment (Janis-Leuenberger) [reply app/helpers/timer_sessions_helper.rb:6]: What about no gap?

**Review by Janis-Leuenberger — APPROVED** [review]

File: app/helpers/timer_sessions_helper.rb (hunk 1 above)
Comment (aneshodza) [reply app/helpers/timer_sessions_helper.rb:6]: Its a `>=`, which means as soon as the gap is one minute it shows. Making it less means everywhere has a gap

File: assets.src/src/styles/time_sessions_table.scss [hunk 2]
```diff
@@ -6,4 +6,8 @@
   .timer-session-table-actions {
     text-align: right !important;
   }
+
+  .gap-marker {
+    border-top: 1px solid red;
+  }
```
Comment (schmijos) [inline assets.src/src/styles/time_sessions_table.scss:12]: Red is the color for errors. Gaps are not errors. Gaps are normal breaks. Please use warn or info colors.

**Review by schmijos — COMMENTED** [review]
See spec here: https://redmine.renuo.ch/issues/19653

Please don't change the gap limit here or address it in a separate ticket. Without proper handling of rounding error in rebalancing a limit of 1 minute doesn't make sense.

**Comment by rnestler** [comment]
@schmijos is this ready to merge?

**Comment by rnestler** [comment]
@aneshodza What's the status?

**Comment by aneshodza** [comment]
@rnestler Josua still needs to review

**Comment by aneshodza** [comment]
I made my own scss vars for the styling of the gap indicator. The issue is, that redmine uses css, which implies that there arent "standard" scss variables.

**Review by schmijos — APPROVED** [review]

**Pull request merged**